lib/redis.js

```diff
@@ -91,8 +91,9 @@ module.exports = CoreObject.extend({
     });
   },
 
-  activeRevision: function(keyPrefix) {
-    var currentKey = keyPrefix + ':current';
+  activeRevision: function(keyPrefix, activationSuffix) {
+    activationSuffix = activationSuffix || 'current';
```

Comment:
I think the right way would be to parametrize those tests so that they don't rely on this default

I'm ok in doing this later, maybe as part of the other refactoring we were discussing?